src/main/java/se/sundsvall/contract/api/model/ContractRequest.java

```diff
@@ -1,15 +1,18 @@
 package se.sundsvall.contract.api.model;
 
+import java.util.List;
+
 import se.sundsvall.dept44.common.validators.annotation.OneOf;
 
+import io.swagger.v3.oas.annotations.media.ArraySchema;
 import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.validation.constraints.NotEmpty;
 import jakarta.validation.constraints.Pattern;
 
 public record ContractRequest(
 	@Schema(description = "Identifier for private person", example = "40f14de9-815d-44a5-a34d-b1d38b628e07") String personId,
 	@Schema(description = "Organization number", example = "771122-1234") String organizationNumber,
-	@Schema(description = "Property designation") String propertyDesignation,
+	@ArraySchema(schema = @Schema(description = "Property designations")) List<String> propertyDesignations,
 	@Schema(description = "External referenceId", example = "123") String externalReferenceId,
 	@Schema(description = "End date (format: yyyy-MM-dd)", example = "2023-01-01") @Pattern(regexp = "(\\d{4})-(\\d{2})-(\\d{2})", message = "Expected date format is yyyy-MM-dd.") String end,
```

Comment:
This might be a `LocalDate` in combination with `@JsonFormat(pattern = "yyyy-MM-dd")` instead of a string. That way the passed date will actually be valid, instead of just conforming to the regexp, without any sanity check on month, day etc

Reply:
Not my doing ;) 
How about. `@DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate end` ?